docs/GUIDE.md

```diff
@@ -0,0 +1,137 @@
+    
+3.  Cocoa Pods
+```Makefile  
+	pod 'AgoraRtcEngine_Special_iOS', '4.2.6.133.VISION'
+```
+## Running the App
+
+Fill in your test mode Agora app ID in the appID field in VisionVideoCallView.swift.
+
+![](/docs/images/blog_avp2_appid.png "appid entry")
+
+Build and run for either the Simulator or your physical VisionPro device. At the same time, prepare a second user to join the chat as a remote user.
+
+### Remote User Setup
+
+Use another device or use the Agora Web demo app for connecting to the video chat channel. The following screenshot shows a web user published video feed using the same App ID and channel name.
+
+![](/docs/images/blog_avp3_remoteuser.png "remote user")
+  
+
+### Running on Simulator
+
+Since there is no camera simulation in the simulator, your local video stream won’t be presented to the other users. However, you can still see other users who have enabled their cameras or custom video feeds on their devices. The following screenshot captures what you will see in the simulator.
+
+
+![](/docs/images/blog_avp4_sim.png "simulator")
+
+### Running on VisionPro Device
+
+Before building your project to the device, Add the following keys to your Info.plist file to request the necessary permissions:
+```xml
+<key>NSCameraUsageDescription</key>
+<string>camera for self capture</string>
+<key>NSMicrophoneUsageDescription</key>
+<string>mic for my voice</string>
+```
+  
+![](/docs/images/blog_avp5_plist.png "plist")
+
+You should confirm your permission to allow the app to use your VisionPro’s camera and microphone. Once you join the channel, you should see yourself in your avatar form and the remote user should show up on your screen!
+
+![](/docs/images/blog_avp6_device.gif "device")
+
+The Web user can also see you in your digital form and you can start chatting together now!
+
+![](/docs/images/blog_avp7_webuser.gif "web user")
+  
+
+## Basic Code Walkthrough
+
+Reading and understanding the code is never a boring task for developers! Here is a break-down of the project source so you can find the area of your interest quickly.
+
+|Class|Purpose|Notes|
+| --------   | -----:  | :----:  |
+Agora_VisionApp|Entry point of application|Standard entry point
+ContentView|UI View presenting to user at the beginning|Accepts the meeting’s channel name. Calls VisionVideoCallView
+VisionVideoCallView|Showing user’s video feeds as a grid format|Input the appID here
+AgoraVideoCanvasView|The video canvas view for rendering the video|This is a reusable class that works on iOS/MacOS as well.
+AgoraManager|The controller of the app business logic that implements Agora RTC Engine’s delegates.|Respond to join channel and leave channel; handles callback events like _didJoinedOfUid_ and _didJoinChannel_.
+
+You should see the code follows the logic flow described in the [official documentation](https://docs.agora.io/en/video-calling/get-started/get-started-sdk?platform=ios).  
```

Comment:
The logic is pretty important because it outlines how the demo is architected to work. I suggest bringing some of this information into the guide to give the user some of the basic understanding and then link out for more information.